paramak/parametric_components/tokamak_plasma_plasmaboundaries.py

```diff
@@ -0,0 +1,209 @@
+import math
+from pathlib import Path
+
+import numpy as np
+import scipy
+from plasmaboundaries import get_separatrix_coordinates
+
+from paramak import Plasma
+
+
+class PlasmaBoundaries(Plasma):
+    """Creates a double null tokamak plasma shape that is controlled
```

Comment:
Can we add something like

Uses the plasma boundaries package to calculate the points